cogs/wordfilter/wordfilter.py

```diff
@@ -550,20 +570,100 @@ async def on_message(self, msg):
     async def on_message_edit(self, msg, newMsg):
         await self.checkWords(msg, newMsg)
 
+    ############################################
+    # COMMANDS - Usage Statistics #
+    ############################################
+    @stat.command(name="rawusage")
+    @commands.guild_only()
+    @checks.mod_or_permissions(manage_messages=True)
+    async def rawCensorUsageList(self, ctx):
+        """
+        displays a raw usage list of all the censored words that have been used
+        """
```

Comment:
Can we follow [PEP257's multiline docstring guideline](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings), and also capitalize the first letter since this gets shown in the helptext?
```suggestion
        """Displays a raw usage list of all the censored words that have been used"""
```